macros/collect_freshness.sql

```diff
@@ -17,10 +17,10 @@
   {% set is_enabled = (enabled_array != []) %}
 
     select
-      {% if is_enabled %}
-      max({{ loaded_at_field }})
+      {% if is_enabled == false %}
```

Comment:
why this change? just wanna make sure there's not an issue with the conditional on line 26 as well